### PR TITLE
Update angularjs.less deprecated selectors

### PR DESCRIPTION
**Pull request by NodeJoSe**
I edit the files based on the emails i received, opened a new pull request because i lost access to the past fork.

I think this is ok, im not sure if i have to add **atom-text-editor** class too

### CODE REVIEW

File: styles/angularjs.less [hunk 1]
```diff
@@ -4,15 +4,14 @@
 // for a full listing of what's available.
 @import "ui-variables";
 
-atom-text-editor,
-atom-text-editor::shadow {
-  .entity.name.tag.block.any.html.angular,
-  .entity.other.attribute-name.html.angular {
+atom-text-editor.editor {
```
Comment (invmatt) [inline styles/angularjs.less]: This should be:

`atom-text-editor {
  atom-text-editor.editor {
`

File: styles/angularjs.less [hunk 2]
```diff
@@ -4,15 +4,14 @@
...
+  .syntax--entity.syntax--other.syntax--attribute-name.syntax--html.syntax--angular {
     color: @text-color-warning;
   }
 
-  .punctuation.definition.block.begin.angular,
-  .punctuation.definition.block.end.angular {
+  .syntax--punctuation.syntax--definition.syntax--block.syntax--begin.syntax--angular,
+  .syntax--punctuation.syntax--definition.syntax--block.syntax--end.syntax--angular {
     color: @text-color-success;
   }
```
Comment (invmatt) [inline styles/angularjs.less]: Need a closing brace as we've added `atom-text-editor.editor {` above.

File: styles/angularjs.less (hunk 1 above)
Comment (invmatt) [inline styles/angularjs.less]: Remove `.editor` and everything else should be good.

Comment (NodeJoSe) [inline styles/angularjs.less]: Removed, thx!

**Comment by outsideris** [comment]
Sorry for late response. Thank you for your PR.

**Pull request merged**